packages/wallet-core/src/tests/fixture.ts

```diff
@@ -0,0 +1,58 @@
+import * as _ from 'lodash';
+
+import {Outcome, StateVariables} from '../types';
+
+export type DeepPartial<T> = {
```

Comment:
Do you think it makes sense to export a test helper from `wallet-core`?

One solution is to `import {fixture} from `src/path/to/fixture`` in `server-wallet`. Normally, this would violate a lint error, but we could ignore that error in one location.

This way,
- we can use the code in `server-wallet` tests
- we don't have to export it from `wallet-core/src/index`.

Reply:
I would probably rather just export the fixture form `wallet-core` unless there are strong reasons not to do so.

Reply:
11aa06a39e31448688042d2a3cfdea13ae0f63c6